compiler/modules/front/codeGen.bal

```diff
@@ -1827,12 +1819,12 @@ function codeGenEqualityExpr(CodeGenContext cx, bir:BasicBlock bb, Environment e
     }
 }
 
-function codeGenVarRefExpr(CodeGenContext cx, s:VarRefExpr ref, Environment env, bir:BasicBlock bb) returns CodeGenError|ExprEffect {
-    if ref.prefix != () {
+function codeGenVarRefExpr(CodeGenContext cx, Environment env, bir:BasicBlock bb, string? prefix, string name, Position startPos) returns CodeGenError|ExprEffect {
```

Comment:
Call the argument `pos`.

Reply:
Fixed with https://github.com/ballerina-platform/nballerina/pull/847/commits/6e54f8f31d724e57d65631c10d6fbf61c368dd7d

Reply:
Fixed with https://github.com/ballerina-platform/nballerina/pull/847/commits/6e54f8f31d724e57d65631c10d6fbf61c368dd7d